api/filrewardsd/pb/filrewards.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package api.filrewardsd.pb;
+option go_package = "github.com/textileio/textile/v2/api/filrewardsd/pb";
+
+import "google/protobuf/timestamp.proto";
+import "api/analyticsd/pb/analytics.proto";
+
+enum RewardType {
+  REWARD_TYPE_UNSPECIFIED = 0;
+  REWARD_TYPE_FIRST_KEY_ACCOUNT_CREATED = 1;
+  REWARD_TYPE_FIRST_KEY_USER_CREATED = 2; // does this fit "register first user?"
+  REWARD_TYPE_FIRST_ORG_CREATED = 3;
+  REWARD_TYPE_INITIAL_BILLING_SETUP = 4;
+  REWARD_TYPE_FIRST_BUCKET_CREATED = 5;
+  REWARD_TYPE_FIRST_BUCKET_ARCHIVE_CREATED = 6;
+  REWARD_TYPE_FIRST_MAILBOX_CREATED = 7;
+  REWARD_TYPE_FIRST_THREAD_DB_CREATED = 8;
+}
+
+message Reward {
+  string org_key = 1;
+  string dev_key = 2;
+  RewardType type = 3;
+  int64 factor = 4;
+  int64 base_nano_fil_reward = 5;
+  google.protobuf.Timestamp created_at = 6;
+}
+
+enum ClaimState {
+  CLAIM_STATE_UNSPECIFIED = 0;
+  CLAIM_STATE_PENDING = 1;
+  CLAIM_STATE_COMPLETE = 2;
+  CLAIM_STATE_FAILED = 3;
+}
```

Comment:
These claim states will be used depending on how the transfer of FIL goes to the target address during a claim.